package.json

```diff
@@ -71,7 +71,7 @@
     "ssim.js": "^3.1.1"
   },
   "peerDependencies": {
-    "jest": ">=20 <=26"
+    "jest": ">=20 <=27"
```

Comment:
Is there a jest 27 yet? I think I'd rather wait to update this until we have a version of jest 27.x.x that we know for a fact has not broken anything jest-image-snapshot uses.

Reply:
Update from 27 to <=26.4.2 (This version exists). Actually, we are using `cypress-image-snapshot` and `ts-jest` dependency in our local package. So, `cypress-image-snapshot` requires `jest-image-snapshot` we have a tool that check peer-dependencies and lets developers know locally if they are any missing peer Dependencies before moving to CI step.
Example: 
`cypress-image-snapshot` ->  `jest-image-snapshot` -> `jest` (with specific condition for peerDeps our tool throws error saying requires ">=20 <=26" version)
`ts-jest` -> `jest` (peerDep required 26.4.0, so we added to our local package.json and tool works fine after adding)

This is the reason we want to create this PR. and then PR for `cypress-image-snapshot`

Reply:
I think the tool you are using is where the issue lies, `>=20 <=26` is a valid semver expression that matches all 26.x.x versions.

Are you able to reproduce the peerDep warning without using that check peer dependencies tool?